File: JavaAssignment-2/GreaterThan10.java
import java.util.*;
class GreaterThan10
{
public static void main(String[] args)
{
int num;
System.out.println("Enter any number");
Scanner sc=new Scanner(System.in);
num=sc.nextInt();
if(num>10)
System.out.println("Greater than 10");
else
System.out.println("Not greater");
}
}

File: C++ Assignments/Assignment 2/Greater than 10.cpp
#include<iostream>
using namespace std;
class Great
{
int num;
public: void display() 
{
cout<<"Enter number \n";
cin>>num;
if(num>10) 
{
cout<<"Greater than 10\n";
}
else
cout<<"Not greater than 10\n";
}
};

int main()
{   
Great obj1;
obj1.display();
return 0;
}